### DISCUSSION TITLE
factorio_rcon.InvalidPassword: The RCON password is incorrect

### DISCUSSION THREAD

**Discussion by KaranTrivedi** [mark9064/factorio-rcon-py#4] [General]
I am using the exact same password as

[  "game_password": "PASS",](url)

in /opt/factorio/config/server-settings.json

Is this the correct setting? Am I doing something wrong?

**Comment by mark9064** [comment]
`"game_password"` is for the password used to join the game in Factorio. To set the RCON password, pass the `--rcon-password` command line flag to Factorio, specifying the password you want after
